### PR TITLE
Added note about --hostname

### PR DESCRIPTION
**Pull request by mbn18**
As discussed [here](https://github.com/Polymer/polymer-cli/issues/300).

### CODE REVIEW

**Comment by mbn18** [comment]
I signed it!

--------------------------------------------------
Michael Ben-Nes - Linux Web Environment Expert.
http://www.epoch.co.il - LinkedIn <http://www.linkedin.com/in/michaelbennes>
Cellular: +972-54-4848113
--------------------------------------------------

File: README.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ Start the development server, and open the default browser:
 
     $ polymer serve -o
 
+By default the server listen to `127.0.0.1`. To listen to a different address add the `--hostname` flag. For example:
```
Comment (justinfagnani) [inline README.md]: "listen" -> "listens"
"add" -> "use"
"`127.0.0.1`" -> "`localhost`"

**Comment by FredKSchott** [comment]
@justinfagnani see #300, current plan is to change default behavior instead of fixing this with documentation

**Comment by mbn18** [comment]
@FredKSchott , Was not sure if you mean Yes or No. Added any way @justinfagnani suggestions

**Comment by FredKSchott** [comment]
Closing per #300, we'd rather change the behavior of serve to use 0.0.0.0 than to document the oddity.

**Comment by FredKSchott** [comment]
just had another issue crop up asking about this, merging to help people for now. If we decide to change this behavior we can update the docs later.

**Comment by FredKSchott** [comment]
Thanks @mbn18!

**Pull request merged**